app/todos/resources.py

```diff
@@ -0,0 +1,85 @@
+# Ask about getting branches up to date once committed
+
+import json
+
+from flask import request
+from flask_restful import Resource
+
+from app import db, Todo
+from app.lib.response_util import buildOkResponse
+
+
+def _returnTodo(todo):
+    """Private method to convert a todo object into a dictionary.
+
+    Args:
+        todo - A todo model.
+    """
+    return {
+            "id": todo.id,
+            "subject": todo.subject
+        }
+
+
+class todoMultiResource(Resource):
+    """Class for creating and accessing todos.
+    """
+    def get(self):  # Should this take a user and return their todos?
```

Comment:
Would you like to implement sessions, cookies, and hashed passwords now, or sort of hack around this by adding a userId query parameter to each request?